cclib/method/bickelhaupt.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Calculation of Bickelhaupt population analysis based on data parsed by cclib."""
+
+import random
+
+import numpy
+
+from cclib.method.population import Population
+
+
+class Bickelhaupt(Population):
+    """Bickelhaupt population analysis."""
+
+    def __init__(self, *args):
+
+        # Call the __init__ method of the superclass.
+        super(Bickelhaupt, self).__init__(logname="Bickelhaupt Population Analysis", *args)
+
+    def __str__(self):
+        """Return a string representation of the object."""
+        return "Bickelhaupt charges of %s" % (self.data)

```

Comment:
Can this (and others) be updated to the new string formatting (e.g. `"Bickelhaupt charges of {}".format(self.data)`)

Reply:
@shivupa Thanks for the comments! I've updated them just now.